docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -0,0 +1,238 @@
+| -------- | ------- | ------------------------------------------ | ---------------------- | -------- |
+| enabled  | boolean | Enable this profile type to be scraped.    | true                   | no       |
+| path     | string  | The path to the profile type on the target. | "/debug/pprof/profile" | no       |
+| delta    | boolean | Whether to scrape the profile as a delta.  | true                   | no       |
+
+### `profile.fgprof`
+
+Collects profiles from an fgprof endpoint.
+
+| Argument | Type    | Description                                | Default           | Required |
+| -------- | ------- | ------------------------------------------ | ----------------- | -------- |
+| enabled  | boolean | Enable this profile type to be scraped.    | false             | no       |
+| path     | string  | The path to the profile type on the target. | "/debug/fgprof"   | no       |
+| delta    | boolean | Whether to scrape the profile as a delta.  | true              | no       |
+
+### `profile.godeltaprof_memory`
+
+Collects profiles from godeltaprof memory endpoint. The delta is computed on the target.
+
+| Argument | Type    | Description                                | Default                  | Required |
+| -------- | ------- | ------------------------------------------ | ------------------------ | -------- |
+| enabled  | boolean | Enable this profile type to be scraped.    | false                    | no       |
+| path     | string  | The path to the profile type on the target. | "/debug/pprof/delta_heap" | no       |
+
+### `profile.godeltaprof_mutex`
+
+Collects profiles from godeltaprof mutex endpoint. The delta is computed on the target.
+
+| Argument | Type    | Description                                | Default                   | Required |
+| -------- | ------- | ------------------------------------------ | ------------------------- | -------- |
+| enabled  | boolean | Enable this profile type to be scraped.    | false                     | no       |
+| path     | string  | The path to the profile type on the target. | "/debug/pprof/delta_mutex" | no       |
+
+### `profile.godeltaprof_block`
+
+Collects profiles from godeltaprof block endpoint. The delta is computed on the target.
+
+| Argument | Type    | Description                                | Default                   | Required |
+| -------- | ------- | ------------------------------------------ | ------------------------- | -------- |
+| enabled  | boolean | Enable this profile type to be scraped.    | false                     | no       |
+| path     | string  | The path to the profile type on the target. | "/debug/pprof/delta_block" | no       |
+
+## Example
+
+The following configuration sets up a scraping job that scrapes two local applications (the Agent itself and Pyroscope). The profiles obtained are then sent over to the receivers as defined by other components.
+
+```river
+pyroscope.scrape "default_settings" {
+  targets    = ["http://localhost:12345"]
```

Comment:
this should be a list of labelsets, not a list of strings, it will not work. Did we copy this example from somewhere?

This should be something like this 
```
targets    = [ { "__address__" = "http://localhost:12345", "service_name" = "example_service" } ]
```

Reply:
I tried to write what I thought worked there based on the other docs apparently it was wrong. Thanks for catching fixed!